dojoup/dojoup

```diff
@@ -295,5 +298,58 @@ banner() {
 '
 }
 
+# Welcome message printed after having installed Dojo.
+welcome_msg() {
+  dojo='\033[1;34m'
+  title='\033[0;32m'
+  emphasis='\033[0;34m'
+  command='\033[0;31m'
+  clear='\033[0m'
+
+  printf "
+═════════════════════════════════════════════════════════════════════════
+
+Ohayo!
+
+Congratulations on successfully installing ${dojo}Dojo${clear} ${DOJOUP_VERSION}! 🎉
+
+Here’s how you can get started:
+
+${title}1. Explore the Dojo Toolkit${clear}
+
+Familiarize yourself with the newly installed components:
+   - ${emphasis}Katana${clear}: Discover how to streamline your development workflow.
+   - ${emphasis}Sozo${clear}: Learn about creating and managing your project environments.
+   - ${emphasis}Torii${clear}: Understand how to connect and interact with various services.
+   - ${emphasis}Dojo Language Server${clear}: Enhance your coding experience with our language server support.
```

Comment:
```suggestion
${title}1. Explore the Dojo Toolkit${clear}

Familiarize yourself with the newly installed components:
   - ${emphasis}Katana${clear}: High performance sequencer for onchain games.
   - ${emphasis}Sozo${clear}: World contract migration planning and execution.
   - ${emphasis}Torii${clear}: Automatic indexer for accessing your world state over gRPC and Graphql.
```

I think the copy here could be improved. Let's remove language server for now since it is borked and will come with scarb soon